src/io/stakater/vc/Git.groovy

```diff
@@ -120,23 +120,22 @@ def tagAndRelease(def versionFile, def repoName, def repoOwner){
   createRelease(version)
 }
 
-def createBinary(def versionFile, def repoName, def repoOwner, String githubToken){
+def createBinary(def versionFile, def repoName, def repoOwner){
   echo "Generating New Version"
   def common = new io.stakater.Common()
   def version = common.shOutput("jx-release-version --gh-owner=${repoOwner} --gh-repository=${repoName} --version-file ${versionFile}")
   sh """
-      echo "${version}" > ${versionFile}
+      echo -n "${version}" > ${versionFile}
   """
   commitChanges(WORKSPACE, "Bump Version to ${version}")
   echo "Pushing Tag ${version} to Git"
   createTagAndPush(WORKSPACE, version)
-  runGoReleaser(WORKSPACE, githubToken)
+  runGoReleaser(WORKSPACE)
 }
 
-def runGoReleaser(String repoDir, String githubToken){
+def runGoReleaser(String repoDir){
```

Comment:
@waseem-h @faizanahmad055 shouldn't this be part of makefiles or this library?

Reply:
Updated as discussed

Reply:
Keeping it as a part of fabric8 pipeline library as it is using other internal methods for git etc